### PR TITLE
Improve test scripts to test che-operator installed via OLM locally

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Improve test scripts to test che-operator installed via OLM locally

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
@flacatus Take a look, if you have a free time.

File: olm/olm.sh [hunk 1]
```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+BASE_DIR=$(cd "$(dirname "$0")" && pwd)
```
Comment (tolusha) [inline olm/olm.sh]: Add license and a description what this script does.

File: olm/README.md [hunk 2]
```diff
@@ -21,13 +21,48 @@ A script is generating new folders/files that can be edited.
...
 ```
 
 Then the changes can be applied in the newly created CSV files.
+
+## Local testing che-operator development version using OLM
+
+To test che-operator with OLM you need to have application registry. You can register on the quay.io and
+use application registry from this service.
+Build your custom che-operator image and push it to the image registry(you also can use quay.io).
+Change `deploy/operator.yaml` image from official to development.
```
Comment (tolusha) [inline olm/README.md]: image -> operator image

File: olm/olm.sh (hunk 1 above)
Comment (AndrienkoAleksandr) [reply olm/olm.sh]: Thanks. Done.

**Pull request merged**